<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scanNow</class>
 <widget class="QWidget" name="scanNow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>70</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Scan Now</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="eventData">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="scanLine"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
